docs/configuration/skills.md

```diff
@@ -0,0 +1,104 @@
+# Skills
+
+## Configuration
+
+- `name`: Unique name for the skill, this will be referenced by other configurations
+- `cooldown_ms`: Time that needs to elapse before the skill is usable again
+- `is_passive`: Marks the skill as a passive skill, this means it can't be triggered. Instead it will trigger on player spawn, so only `GiveEffect` makes sense for it
+- `mechanics`: Core mechanic of the skill (e.g hit, shoot, etc)
+
+### Mechanics
+
+This are the mechanics so far. Every mechanic will add a configuration field with the same name of it which is a nested object with configuration attributes specific for it
+
+- `GiveEffect`: This makes the skill give a certain effect
```

Comment:
if this is intended to only work with the instigator (i.e. it doesn't receive targets) maybe I'd consider a name like `GiveEffectSelf`